estate/models/estate_property.py

```diff
@@ -0,0 +1,22 @@
+from odoo import fields, models
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate App"
+
+    name = fields.Char(required=True)
+    description = fields.Text() 
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
```

Comment:
```suggestion
    is_garage = fields.Boolean()
    is_garden = fields.Boolean()
```
Try to define boolean fields in this way which will be easier to use and understand in code as well.